c/driver/common/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+add_library(adbc_driver_common STATIC utils.c)
+include_directories(SYSTEM ${REPOSITORY_ROOT})
```

Comment:
All(?) references to REPOSITORY_ROOT can be replaced with a proper CMake variable (CMAKE_SOURCE_DIR I think)

Reply:
Looking at this more closely the `REPOSITORY_ROOT` is still needed one way or another when including adbc.h, since that technically lives outside of the /c source directory. 

Happy to do something here but I think better in a follow up - otherwise might make these changes confusing

Reply:
Ah, ok - then let's not worry about it